api/libproject/devstart.rb

```diff
@@ -1286,12 +1286,18 @@ def create_auth_domain(cmd_name, args)
   op.add_option(
     "--project [project]",
     ->(opts, v) { opts.project = v},
-    "Project to create the authorization domain"
+    "Workbench Project (environment) for creating the authorization domain"
   )
+  op.add_option(
+    "--user [user]",
```

Comment:
nit: the other similar usage we have in project.rb calls this flag "account" https://github.com/all-of-us/workbench/blob/ee18c9c28ae92fb1e11a6d746731d4361cfe9b5c/api/libproject/devstart.rb#L1398

Reply:
In this case we actually do want a RW user, not a Service Account.